lib/shipit.rb

```diff
@@ -101,8 +103,19 @@ def redis
     )
   end
 
-  def github
-    @github ||= GitHubApp.new(secrets.github)
+  def github(organization: github_default_organization)
+    org = organization.to_sym
+    raise GithubOrganizationUnknown if secrets.github[org].nil?
+    @github ||= {}
+    @github[organization] ||= GitHubApp.new(organization, secrets.github[org])
+  end
+
+  def github_default_organization
+    secrets.github.keys.first
```

Comment:
This requires the current structure of the secrets to be slightly remodelled, right?
Looking at [the development secrets example above](https://github.com/Shopify/shipit-engine/pull/1151/files#diff-d077374229ea196a27874f1d46a8c013dce1af4ac5ac7c4ca0ba74bc09f2312aR5-R22), the default app needs to be nested one level deeper now, I think

Reply:
New config can now look like this. However, this branch now supports backward capability for the old-style config.

```yaml
github:
    org1:
      app_id: 
      installation_id:
      webhook_secret:
      private_key:
      oauth:
        id: 
        secret: 
        teams:
    org2:
      app_id: 
      installation_id:
      webhook_secret:
      private_key: 
      oauth:
        id: 
        secret: 
        teams:
```